scripts/release.sh

```diff
@@ -89,16 +89,28 @@ merge_develop() {
     echo -e "Successfully merged into develop branch\n"
 }
 
+rename_draft_release() {
+    release_id=$(
+        command curl -H "Authorization: token ${GITHUB_TOKEN}" -X GET \
+            https://api.github.com/repos/airyhq/airy/releases | jq '.[0].id'
+    )
+    command curl -H "Authorization: token ${GITHUB_TOKEN}" -X PATCH \
+        https://api.github.com/repos/airyhq/airy/releases/"$release_id" \
+        -d "{\"name\":\"${release_number}\", \"tag_name\":\"${release_number}}"
```

Comment:
This will actually create a new tag on the default branch of the repo. Because when we run this the tag won't be there yet. 
<img width="661" alt="Screenshot 2021-03-26 at 12 47 02" src="https://user-images.githubusercontent.com/54705263/112627066-655bf880-8e31-11eb-808c-13adf66dc375.png">

Reply:
not sure what the full implications of this are. Should we remove `git create tag` and just keep this here in the curl?